### DISCUSSION TITLE
Can I customize the prompt for ios requesting permission?

### DISCUSSION THREAD

**Discussion by cnhuz** [invertase/react-native-firebase#6503] [Q&A]
For permission requests mentioned in this document https://rnfirebase.io/messaging/ios-permissions
`await messaging().requestPermission();`

When my ios app was put on the shelves for review, it was rejected. When my system language is Chinese and the app language is Uzbek, the pop-up prompt is as shown below. It is in English. Can I modify the pop-up content? If so, which part of the content (1234) can I modify? How to modify it?

![image](https://user-images.githubusercontent.com/48916521/187812354-d0c6ba60-48a4-423c-9443-ade4665cde42.png)

**Comment by mikehardy** [comment]
I think you will want to use react-native-permissions for real permissions management. The API here is little more than a stub
I don't think your localization issues will be fixed by that though. That looks like the standard English one and I think that may be related to your Xcode project configuration

**Reply by cnhuz** [reply]
Thank you, I will learn about react-native-permissions
